docs/learn/prompts.md

```diff
@@ -125,33 +97,29 @@ If there are any other such specifiers you would find useful, let us know!
 
 By default, Mirascope treats the entire prompt as a single user message. However, you can use the `SYSTEM`, `USER`, and `ASSISTANT` keywords to specify different message roles, which we will parse into `BaseMessageParam` instances:
 
-```python
-from mirascope.core import BasePrompt, prompt_template
-
-
+```python hl_lines="3 4 11 12"
 @prompt_template(
     """
     SYSTEM: You are the world's greatest librarian
     USER: Recommend a {genre} book
     """
 )
-class RecommendBookPrompt(BasePrompt):
-    genre: str
+def book_recommendation_prompt(genre: str): ...
 
-
-prompt = RecommendBookPrompt(genre="fantasy")
-print(prompt)
-# > SYSTEM: You are the world's greatest librarian
-#   USER: Recommend a fantasy book
-print(prompt.message_params())
-# > [BaseMessageParam(role='system', content="You are the world's greatest librarian"), BaseMessageParam(role='user', content='Recommend a fantasy book')]
+print(book_recommendation_prompt("fantasy"))
+# > [
+#     BaseMessageParam(role='system', content="You are the world's greatest librarian"),
+#     BaseMessageParam(role='user', content='Recommend a fantasy book')
+#   ]
 ```
 
+In this example, we use the `SYSTEM` and `USER` keywords to specify different roles for parts of the prompt. The `message_params()` method returns a list of BaseMessageParam instances, each representing a different role and its content.
```

Comment:
this is stale as we're no longer showing `message_params()` and are instead just calling the prompt template function